pages/common/ts-node.md

```diff
@@ -0,0 +1,28 @@
+# ts-node
+
+> Run TypeScript code directly, without any compiling.
+> More information: <https://www.npmjs.com/package/ts-node>.
+
+- Execute a TypeScript file without compiling (`node` + `tsc`).
```

Comment:
```suggestion
- Execute a TypeScript file without compiling (`node` + `tsc`):
```

Reply:
This is what the bot is complaining about @SpiderMath 

Can you please fix it?